app/models/header_lookup.rb

```diff
@@ -37,7 +37,7 @@ def pid_lookup_by_field(term="", field="")
       searchable_file = @@searchable_lcsh_file
       memoized_terms = @@memoized_lcsh
       memoized_file = MEMOIZED_LCSH_FILE
-    elsif field == :subject_name || field == :subject_geographic || field == :lcnaf
+    elsif field == :subject_name || field == :subject_geographic
```

Comment:
There is no field "lcnaf" on the model GenericFile so this check will never be true. I think this was left behind by accident when this function was refactored previously.